deposit/src/main/java/edu/unc/lib/deposit/fcrepo4/IngestContentObjectsJob.java

```diff
@@ -560,12 +576,32 @@ private void addDescription(ContentObject obj) throws IOException {
 	}
 	
 	private void addPremisEvents(ContentObject obj) {
-		File premisFile = new File(getEventsDirectory(), obj.getPid().getUUID() + ".xml");
+		File premisFile = new File(getEventsDirectory(), obj.getPid().getUUID() + ".ttl");
 		if (!premisFile.exists()) {
 			return;
 		}
-		FilePremisLogger premisLogger = new FilePremisLogger(obj.getPid(), premisFile, repository);
+		PremisLogger premisLogger = getPremisLogger(obj.getPid());
 		List<PremisEventObject> events = premisLogger.getEvents();
+		// add premis events to fedora
 		obj.addPremisEvents(events);
 	}
+	
+	private void addIngestionEvent(ContentObject obj) throws IOException {
```

Comment:
This method looks pretty good.  Looking at the spreadsheet a little more though I think there may need to be a some changes to the functionality, which is my fault for not working more through the requirements:

1) I don't think this works correctly for the ingestFolder case because the children are added after premis is uploaded, so obj.getMembers() wouldn't return anything.  There doesn't appear to be an ingestFolder integration test at the moment except the resume case.  But this ties in with another suggestion
2) Based on the spreadsheet, I think when a container is ingested in should have two details: "ingested as PID: <>" referring to itself, and "added child object <>" for the items added to it.  If no children are added, then I don't think it needs the second message
3) The "added child object <>" message should also be added to the existing container where the deposit was ingested to, since it records how many items were added to the existing container.

It may be helpful to split this method into two methods, one which records the ingestion event info for the object itself, and one that records ingestion of children.  These can be two separate events.

Also, in the case of adding to an existing container, obj.getMembers() wouldn't be accurate since there may already be existing objects in the container.